engine/database/testabilities/fixture_database.go

```diff
@@ -0,0 +1,71 @@
+package testabilities
+
+import (
+	"testing"
+
+	"github.com/bitcoin-sv/spv-wallet/engine/database"
+	testengine "github.com/bitcoin-sv/spv-wallet/engine/testabilities"
+	"github.com/bitcoin-sv/spv-wallet/models/bsv"
+	"github.com/stretchr/testify/require"
+	"gorm.io/gorm"
+)
+
+type DatabaseFixture interface {
+	DB() DatabaseDataFixtures
+	GormDB() *gorm.DB
```

Comment:
What do you think about having the GormDB() inside DatabaseDataFixtures, maybe change it's name to DatabaseFixtures and inside there we could have an Instance() method and access the db like that? 

I think having two objects name similar like DB() and GormDB() can be a little confusing. WDYT?

Reply:
It confused me when I was reading the code, but now that I tried to write a simple implementation of what I had in mind it's not a good idea to do it like this. I mixed up what the `DatabaseDataFixtures` actually do with what I thought it should do. We can discuss on priv but it was my mistake. So please ignore this comment or maybe just consider changing `GormDB()` to `DBInstance()` or `Instance()`. But if you think it doesn't give much value it's still fine.